sympy/core/tests/test_power.py

```diff
@@ -526,3 +526,11 @@ def test_issue_14815():
 def test_issue_18509():
     assert unchanged(Mul, oo, 1/pi**oo)
     assert (1/pi**oo).is_extended_positive == False
+
+
+def test_issue_18762():
+    from sympy.abc import psi, epsilon
+    g0 = Symbol('g')
+    g0 = sqrt(1 + (epsilon)**2 - 2*(epsilon)*cos(psi))
+    assert str(g0.series(x = epsilon, x0 = 1, n = 3)) == '(epsilon - 1)*(-cos(psi)/sqrt(2 - 2*cos(psi)) + 1/sqrt(2 - 2*cos(psi))) + (epsilon - 1)**2*(-sqrt(2 - 2*cos(psi))*cos(psi)**2/(2*(4*cos(psi)**2 - 8*cos(psi) + 4)) + sqrt(2 - 2*cos(psi))*cos(psi)/(4*cos(psi)**2 - 8*cos(psi) + 4) - sqrt(2 - 2*cos(psi))/(2*(4*cos(psi)**2 - 8*cos(psi) + 4)) + 1/(2*sqrt(2 - 2*cos(psi)))) + sqrt(2 - 2*cos(psi)) + O((epsilon - 1)**3, (epsilon, 1))'
```

Comment:
Why `str`? Cannot this be

    assert g0.series(e, 1, 3) == (e - 1)*...

possibly continued on another line?